### PR TITLE
feat: updated xdefi icon

### PR DESCRIPTION
**Pull request by j-acks0n**
### Description

- Updated xdefi icon
- Updated READM.md for out of date doc link

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [ ] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

File: packages/xdefi/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/xdefi",
-  "version": "2.0.4",
+  "version": "2.0.5",
```
Comment (leightkt) [inline packages/xdefi/package.json:3]:
```suggestion
  "version": "2.0.5-alpha.1",
```

**Comment by leightkt** [comment]
@j-acks0n Please update to alpha.1 version and also update the @web3-onboard/xdefi version in docs/package.json. Thx!

File: packages/demo/package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "@web3-onboard/venly": "^2.0.0",
     "@web3-onboard/walletconnect": "^2.4.5-alpha.1",
     "@web3-onboard/web3auth": "^2.2.2",
-    "@web3-onboard/xdefi": "^2.0.3",
+    "@web3-onboard/xdefi": "^2.0.5",
```
Comment (leightkt) [inline packages/demo/package.json:58]:
```suggestion
    "@web3-onboard/xdefi": "^2.0.5-alpha.1",
```

**Comment by taylorjdawson** [comment]
@j-acks0n along with @leightkt suggestion can you also merge latest develop? thanks!

**Review by leightkt — APPROVED** [review]

**Pull request merged**